src/styles/styles.js

```diff
@@ -1195,7 +1196,12 @@ const styles = {
         ...{borderRadius: variables.componentBorderRadiusSmall},
     },
 
-    reportTransaction: {
+    reportTransactionCancel: {
```

Comment:
This is a generic style that isn't specific to the cancel button. I suggest using helper styles for this: `flexRow` and you'll need to add one for `justifyContentStart` (in flex.js)

Reply:
Ah, that's much better -- thanks!